source/develop/release-management/features/network/import-vm-net-changes.html.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Change network information in VM import
+category: feature
+authors: yevgenyz
+wiki_category: Feature
+wiki_title: Features/Change network information in VM import
+wiki_revision_count: 1
+wiki_last_updated: 2016-08-29
+feature_name: Change network information in VM import
+feature_modules: engine/api-model
+feature_status: Work in progress for oVirt 4.1
+---
+
+# Change network information in VM import
+
+## Summary
+
+Currently importing a VM from an external source might result in the 
+non-functional VM from the network perspective:
+
+* vNics that are not connected to any network that is defined in the 
+oVirt the VM was imported to or is connected to an undesired vNic 
+profile.
```

Comment:
This is slightly complicated. Can we split this in two:
* vNics not connected to any network defined in oVirt
* vNics connected to an undesired vNic profile